api/src/repositories/users.ts

```diff
@@ -0,0 +1,18 @@
+import { prisma } from "../lib/prisma";
+import { LoginInput } from "../validators/login-form";
+
+class UsersRepository {
+    async find({ email }: LoginInput) {
```

Comment:
```suggestion
    async find(email:  string) {
```

Aqui a gente muda isso pq não vamos precisar de nenhuma outra prop além de email então não faz sentido mandar como objeto.

Reply:
Ok